mlrun/datastore/datastore_profile.py

```diff
@@ -193,6 +196,10 @@ class DatastoreProfileKafkaSource(DatastoreProfile):
     kwargs_public: typing.Optional[dict]
     kwargs_private: typing.Optional[dict]
 
+    def get_topic(self) -> str:
```

Comment:
Should be changed to:
```suggestion
    def get_topic(self) -> typing.Optional[str]:
```
According to your comment https://github.com/mlrun/mlrun/pull/7301#discussion_r1966534591